static/phyui/manual_clustering.js

```diff
@@ -104,10 +104,25 @@ define(function (require) {
 
     events.on('kernel_connected.Kernel', function(){
         console.log('session done');
-        myhack.create_result_cell('#placeholder1', "from phyui.session import start_manual_clustering; session = start_manual_clustering('/home/ctaf/src/cortex/data/test_hybrid_120sec.kwik', backend='ipynb_webgl')");
-        myhack.create_result_cell('#placeholder2', "import vispy; \
-                                                    vispy.app.use_app('ipynb_webgl'); \
-                                                    w = session.show_waveforms()");
+        myhack.create_result_cell('#placeholder1', "from phyui.session import start_manual_clustering; \
+                                                    session = start_manual_clustering('/home/ctaf/src/cortex/data/test_hybrid_120sec.kwik', backend='ipynb_webgl')");
+        var cc = myhack.create_result_cell('#placeholder2', "import vispy; \
+                                                             vispy.app.use_app('ipynb_webgl'); \
+                                                             w = session.show_waveforms()");
```

Comment:
I hope we can find something less ugly in the future! but I assume this is just a prototype :p